pac-kube-config-files/nginx-config.yml

```diff
@@ -0,0 +1,7 @@
+
+apiVersion: v1
+data:
+  proxy-set-headers: default/custom-headers
```

Comment:
I'm not really clear on how that maps to `nginx-custom-headers.yml` namespace-wise, but the rest looks OK.